packages/block-library/src/video/tracks-editor.js

```diff
@@ -58,8 +58,7 @@ function TrackList( { tracks, onEditPress } ) {
 				>
 					<span>{ track.label } </span>
 					<Button
-						// TODO: Switch to `true` (40px size) if possible
-						__next40pxDefaultSize={ false }
+						__next40pxDefaultSize
```

Comment:
This can be seen when you add `core/video` block and add a track list and check for the `edit` button.

| Before | After     |
| :-------- | :------- |
| <img width="404" alt="Edit-track-before" src="https://github.com/user-attachments/assets/8ecd9395-3ac1-427c-b9cc-966fec2f86c7"> | <img width="404" alt="Edit-track-after" src="https://github.com/user-attachments/assets/fa996d37-135d-452e-8e1d-c676840eb2de"> |